src/lib.rs

```diff
@@ -360,15 +358,43 @@ impl Game {
             self.activate_near(position)?;
         }
 
-        let player: &Player = self.local_player().unwrap();
-        self.send_msg(&Message::Move(MoveMessage {
-            color: player.color,
-            inputs: player.inputs,
-            current_position: player.position,
-        }))?;
+        let speed_changed: bool;
+        {
```

Comment:
What is the significance of the block here? Just a closed in variable scope?

Reply:
Yeah, and a way to be super clear that the mutable `player` inside this block should live no longer than this block.

In older versions of Rust you had to do this kind of inline block a lot more often, but it's still sometimes necessary